pkg/distributor/distributor.go

```diff
@@ -707,7 +714,7 @@ func (d *Distributor) Push(ctx context.Context, req *cortexpb.WriteRequest) (*co
 	}
 
 	d.receivedSamples.WithLabelValues(userID).Add(float64(validatedSamples))
-	d.receivedExemplars.WithLabelValues(userID).Add((float64(validatedExemplars)))
+	d.receivedExemplars.WithLabelValues(userID).Add(float64(validatedExemplars))
```

Comment:
Ideally, don't make unrelated changes.

Reply:
Reverted it. Just was a low-hanging fruit and didn't see evil, but I get it generates noise.